packages/shared/src/components/modals/NewSourceModal.tsx

```diff
@@ -245,6 +246,14 @@ export default function NewSourceModal(props: ModalProps): ReactElement {
         Have an idea for a new source? Insert its link below to add it to the
         feed.
       </p>
+      <a
+        className="font-bold typo-callout text-theme-label-link"
```

Comment:
I just realized you removed the `underline` class. It should be part of the design.
```suggestion
        className="font-bold underline typo-callout text-theme-label-link"
```